styles/main.css

```diff
@@ -376,6 +423,172 @@ form .cta-button:hover {
   border-style: solid;
   border-width: 2px;
   border-radius: 5px;
-  margin: 15px 100px;
+  margin: 30px 30%;
+  margin-bottom: 0;
   text-align: center;
 }
+
+/* DESKTOP DESIGN */
+
+@media screen and (min-width: 768px) {
```

Comment:
- [ ] As stated in the requirements, kindly use min-width in the range between 992px and 1100px, since most desktop screens fall above 992px in width.

![image](https://user-images.githubusercontent.com/61377322/145517377-1be287eb-79f0-456c-995a-936535ef2518.png)

Reply:
Done, thank you! :-)